win32ss/user/user32/windows/mdi.c

```diff
@@ -1058,7 +1058,7 @@ static void MDI_UpdateFrameText( HWND frame, HWND hClient, BOOL repaint, LPCWSTR
 
 	    static const WCHAR lpBracket[]  = {' ','-',' ','[',0};
 	    static const WCHAR lpBracket2[]  = {']',0};
-	    int	i_frame_text_length = strlenW(ci->frameTitle);
+	    int	i_frame_text_length = lstrlenW(ci->frameTitle);
```

Comment:
Not sure what this change is about. `wcslen` is usually preferable to lstrlenW though.

Reply:
wcslen returns size_t, lstrlenW returns int.

Reply:
I would say that's a reason to change the variable type, not the function. lstrlen hides bugs by using SEH.